cmd/acr/purge.go

```diff
@@ -162,6 +165,7 @@ func newPurgeCmd(rootParams *rootParameters) *cobra.Command {
 	cmd.Flags().StringArrayVarP(&purgeParams.configs, "config", "c", nil, "Authentication config paths (e.g. C://Users/docker/config.json)")
 	cmd.Flags().Int64Var(&purgeParams.filterTimeout, "filter-timeout-seconds", defaultRegexpMatchTimeoutSeconds, "This limits the evaluation of the regex filter, and will return a timeout error if this duration is exceeded during a single evaluation. If written incorrectly a regexp filter with backtracking can result in an infinite loop.")
 	cmd.Flags().IntVar(&purgeParams.concurrency, "concurrency", defaultPoolSize, concurrencyDescription)
+	cmd.Flags().Int32Var(&purgeParams.repoBatchSize, "repository-batch-size", defaultRepoBatchSize, repoBatchSizeDescription)
```

Comment:
nit: just wondering if something like `repository-page-size` would be a better name, Since the term "page" is more commonly used?